hazelcast-client/src/test/java/com/hazelcast/client/ClientOutBoundPortTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hazelcast.client;
+
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.test.ClientTestSupport;
+import com.hazelcast.client.test.TestHazelcastFactory;
+import com.hazelcast.config.XmlConfigBuilder;
+import com.hazelcast.core.Client;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.spi.discovery.multicast.MulticastDiscoveryStrategy;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.TestEnvironment;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.io.InputStream;
+import java.util.Arrays;
+
+/**
+ * Created by gokhan oner on 6/21/17.
+ */
+@RunWith(HazelcastParallelClassRunner.class)
```

Comment:
it should not use the parallel runner when a real network is configured. right now there is no difference as there is only a single test-case. but it's too easy to add a new case and forget to change the runner.